paasta_tools/cli/cmds/status.py

```diff
@@ -2587,6 +2592,7 @@ def _use_new_paasta_status(args, system_paasta_config) -> bool:
     tron=print_tron_status,
     adhoc=print_adhoc_status,
     flink=print_flink_status,
+    flinkeks=print_flink_status,
```

Comment:
I think print_flink_status is still communicating with the wrong client, either we need to act and get the correct client based on the instance type, or pass the client to the function